cli/ballerina-cli/src/main/resources/cli-help/ballerina-tool.help

```diff
@@ -33,16 +35,22 @@ EXAMPLES
        Pull a specific version of a tool from Ballerina Central.
            $ bal tool pull openapi:1.5.0
 
-       List all the tools available locally.
-           $ bal tool list
-
-       Search Ballerina Central for tools.
-           $ bal tool search openapi
-
        Remove a specific version of previously pulled tool.
            $ bal tool remove openapi:1.5.0
 
        Remove all the versions of a previously pulled tool.
            $ bal tool remove openapi
 
+       Update a tool to the latest patch version.
+           $ bal tool update openapi
```

Comment:
Let's not use openapi in the example since its not shipped as a tool yet. Use edi or health